tests/framework/extensions/provisioninginput/config.go

```diff
@@ -9,6 +9,7 @@ import (
 
 type Version string
 type PSACT string
+type SshTestCase string
```

Comment:
nit; change this to `SSHTestCase`.

Reply:
Yeah this one is my bad, I suggested it here: https://github.com/rancher/rancher/pull/42854#discussion_r1329452626 but abbreviations (which technically ssh falls under that category) should always have all letters capitalized in exported types/fields and lowercase in non exported types/fields.